src/test/regress/sql/multi_extension.sql

```diff
@@ -78,6 +78,71 @@ ALTER EXTENSION citus UPDATE TO '6.1-15';
 ALTER EXTENSION citus UPDATE TO '6.1-16';
 ALTER EXTENSION citus UPDATE TO '6.1-17';
 ALTER EXTENSION citus UPDATE TO '6.2-1';
+
+-- Test non-distributed queries work even in version mismatch
```

Comment:
(in fact that's where I think it should live after this is released, otherwise this'll be a bit weird as more version upgrade scripts are added and whatnot)